### PR TITLE
doc: add missed word in japan/attendee-prerequisites.md

### PR DESCRIPTION
**Pull request by NT-marlowe**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
add a missed word in japan/attendee-prerequisites.md

**Which issue(s) this PR fixes**:
NONE

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
NONE

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: japan/assets/attendee-prerequisites.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Kubernetesコミュニティでは2022年3月頃にCLAの手順がEasyCLAに移
 
 サインアップを一度実施すると、以降のKubernetesコミュニティでの活動で再度実施する必要はありません。  
 1. PullRequestを作成すると linux-foundation-easycla から以下のようにコメントされます。  
-  `Please here to be authorized` をクリックします。  
+  `Please click here to be authorized` をクリックします。
```
Comment (nasa9084) [inline japan/assets/attendee-prerequisites.md:57]:
```suggestion
  `Please click here to be authorized` をクリックします。  
```

行末にスペース二つ　はmarkdownの仕様で改行を示すはずなので消しちゃダメな奴だと思います！多分！

Comment (NT-marlowe) [reply japan/assets/attendee-prerequisites.md]: @nasa9084 
ご指摘ありがとうございます 🙇 修正しました！

**Comment by shu-mutou** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**